src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/RequestSql/SqlRequestFormDataProvider.php

```diff
@@ -146,15 +146,15 @@ private function getEditRequestSqlCommand(array $requestSqlData)
     /**
      * Transform exception into translatable errors.
      *
-     * @param SqlRequestException $e
+     * @param SqlRequestConstraintException|SqlRequestException $e
      *
      * @return array Errors
      */
     private function handleException(SqlRequestException $e)
     {
         $exceptionType = get_class($e);
```

Comment:
Then `$exceptionType` is now useless and should be removed I'm surprised PHPStan doesn't say anything about unused variable Or is it in another level?